### DISCUSSION TITLE
Wierd line above tab-bar

### DISCUSSION THREAD

**Discussion by kskruffnekk** [wezterm/wezterm#2193] [Q&A]
Hello there, 
I'm new to wezterm but so far I really like it. However, I have a wierd purple line above the tab-bar:

![grafik](https://user-images.githubusercontent.com/108282882/175960731-cd70cbcb-94bb-4d39-9512-0df09d4a7bfa.png)

What is that, how can I remove it / edit the color.

I use Fedora 36 with Wayland and zsh.

For the record: It stays purple, no matter which color scheme I use...

**Comment by wez** [comment]
I think that's the titlebar bottom color specified over here:
https://wezfurlong.org/wezterm/config/lua/config/window_frame.html

**Reply by kskruffnekk** [reply]
Thank you very much, that's it!

Can you tell me why the +-icon is looking wierd?

![grafik](https://user-images.githubusercontent.com/108282882/176019837-f7e95f8c-7144-4e34-89d7-bfd2b5f1e826.png)
